nautobot/extras/tests/test_views.py

```diff
@@ -909,3 +912,104 @@ def test_create_object_with_constrained_permission(self):
         for cf in CustomField.objects.all():
             cf.delete()
         super().test_create_object_with_constrained_permission()
+
+
+class ScheduledJobTestCase(
+    ViewTestCases.ListObjectsViewTestCase,
```

Comment:
Since we have `ScheduledJobView` and `ScheduledJobDeleteView` classes, should we also include `GetObjectViewTestCase` and `DeleteObjectViewTestCase` here?